src/components/spinner/DsSpinner.less

```diff
@@ -1,21 +1,18 @@
+@import (reference) './ds-spinner';
+
 :host {
+  &:extend(ds-spinner);
+
   animation: spin 0.8s linear infinite;
   animation-play-state: running;
   border-color: transparent transparent currentColor currentColor;
   border-radius: 1em;
   border-style: solid;
   border-width: 1px;
-  display: inline-block;
-  height: 1em;
-  width: 1em;
 
   &([paused]) {
     animation-play-state: paused;
   }
-
-  &([hidden]) {
```

Comment:
Removed, because all supported browsers support the `hidden` attribute (IE11 + Evergreen).